### PR TITLE
Feature : Added the feature of sending the notification of latest video/note to discord

### PR DESCRIPTION
**Pull request by amanbairagi30**
### **Problem Statement**

Currently, whenever a new video is uploaded into the CMS, Harkirat has to update the Discord community about it. This process requires additional effort after posting the video in the CMS. 

### **Solution**

I have implemented a solution where whenever a new video or note is added from the CMS admin panel, a notification for that video or note can be seen in Discord. I have made this an optional choice. If moderators want to add a video/note without sending a notification to discord, they can do so. Additionally , I have provided a checkbox for this purpose . Along with that in discord message you can even go to that particular video /note location by clicking the link provided there .

You can see the changes below : 

https://github.com/code100x/cms/assets/118182376/718a7f68-3209-468d-868c-11362fda194d

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by TanmayDhobale** [comment]
damn @amanbairagi30

**Comment by amanbairagi30** [comment]
@hkirat Please review this PR .

**Comment by ichiragkumar** [comment]
great job, it was on my list but you did this

File: src/components/admin/AddContent.tsx [hunk 1]
```diff
@@ -2,43 +2,142 @@
...
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const respData: any = await response.json();
+
+    const data = {
+      type,
+      thumbnail: imageUri,
```
Comment (hkirat) [inline src/components/admin/AddContent.tsx:76]: hardcode this to https://d2szwvl7yo497w.cloudfront.net/courseThumbnails/video.png

File: src/components/admin/AddContent.tsx [hunk 2]
```diff
@@ -2,43 +2,142 @@
...
+    const fieldsToCheck = [
+      'video_1080p_0',
+      'video_720p_0',
+      'video_360p_0',
+      'notionId',
+    ];
+
+    setLoading(true);
+
+    const response = await fetch('/api/admin/content', {
```
Comment (hkirat) [inline src/components/admin/AddContent.tsx:55]: Should only happen if this is a video/notion doc
shouldnt' happen for folders

**Comment by amanbairagi30** [comment]
Closing because of some weird merge conflicts and raised a new PR at #601

**Comment by amanbairagi30** [comment]
Closing because of some weird merge conflicts and raised a new PR at #601

File: src/components/admin/AddContent.tsx (hunk 1 above)
Comment (amanbairagi30) [reply src/components/admin/AddContent.tsx:76]: @hkirat , Please see #601 , I have raised a new PR and resolved all of the changes you suggested in live

File: src/components/admin/AddContent.tsx (hunk 2 above)
Comment (amanbairagi30) [reply src/components/admin/AddContent.tsx:55]: @hkirat , Please see #601 , I have raised a new PR and resolved all of the changes you suggested in live

**Pull request closed without merging**